src/OAM.md

```diff
@@ -1,45 +1,49 @@
 
-# VRAM Sprite Attribute Table (OAM)
+# Object Attribute Memory (OAM)
 
-The Game Boy PPU can display up to 40 sprites either in 8x8 or
-in 8x16 pixels. Because of a limitation of hardware, only ten sprites
-can be displayed per scan line. Sprite tiles have the same format as
-BG tiles, but they are taken from the Sprite Tiles Table located at
+The Game Boy PPU can display up to 40 movable objects (or sprites), each 8x8 or
+8x16 pixels. Because of a limitation of hardware, only ten objects
+can be displayed per scanline. Object tiles have the same format as
+BG tiles, but they are taken from tile blocks 0 and 1 located at
 $8000-8FFF and have unsigned numbering.
 
-Sprite attributes reside in the Sprite Attribute Table (OAM: Object
-Attribute Memory) at \$FE00-FE9F. Each of the 40 entries consists of
+Object attributes reside in the object attribute memory (OAM) at \$FE00-FE9F.
+(This corresponds to the sprite attribute table on a TMS9918 VDP.)
```

Comment:
People familiar with the previous version of Pan Docs or arriving through an inbound link might be searching the document for the phrase "sprite attribute table". So might people coming from the MSX, ColecoVision, or Sega 8/16-bit scene. So that's what I decided to spend my one "sprite" per page on for this page.

Reply:
Okay, fair enough.